opsgenie/resource_opsgenie_team_membership.go

```diff
@@ -0,0 +1,159 @@
+package opsgenie
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/opsgenie/opsgenie-go-sdk-v2/team"
+	"log"
+	"strings"
+)
+
+func resourceOpsGenieTeamMembership() *schema.Resource { //TODO encode the e-mail addrs (e.g. because of +)? https://github.com/opsgenie/opsgenie-go-sdk-v2/issues/62
+	return &schema.Resource{
+		Create: resourceOpsGenieTeamMembershipCreate,
+		Read:   handleNonExistentResource(resourceOpsGenieTeamMembershipRead),
+		//Update: resourceOpsGenieTeamMembershipUpdate, // requires https://github.com/opsgenie/opsgenie-go-sdk-v2/issues/59
```

Comment:
I didn't quite understand the issue in go-sdk can you describe the issue

Reply:
There's no way to update a member ship with go-sdk, that's why don't support it here too. Updating a membership would only involve changing a role without removing+adding the user from the team.